src/components/Quiz/QuizWidget.tsx

```diff
@@ -243,25 +319,32 @@ const QuizWidget: React.FC<IProps> = ({ quizKey, maxQuestions }) => {
     )
   }
 
-  // TODO: Allow user to submit quiz for storage
-  // TODO: Fix a11y keyboard tab stops
-
   // Render QuizWidget component
   return (
-    <Flex as="aside" width="full" direction="column" alignItems="center">
-      <Heading
-        as="h2"
-        mb={12}
-        textAlign="center"
-        scrollBehavior="smooth"
-        scrollMarginTop={24}
-        id="quiz"
-      >
-        <Translation id="test-your-knowledge" />
-      </Heading>
-      <Box
+    <Flex width="full" direction="column" alignItems="center">
+      {/* Hide heading if quiz is not in Learning Quizzes Hub page */}
+      {isStandaloneQuiz && (
+        <Heading
+          as="h2"
+          mb={12}
+          textAlign="center"
+          scrollBehavior="smooth"
+          scrollMarginTop={24}
+          id="quiz"
+        >
+          <Translation id="test-your-knowledge" />
+        </Heading>
+      )}
+
+      <Stack
         w="full"
         maxW="600px"
+        h={isStandaloneQuiz ? "100%" : { base: "100vh", md: "100%" }}
```

Comment:
I believe we're causing issues on iOS by miscalculating the size of the screen when the navbar is present by using `100vh` here: [See Nuno's comment here](https://github.com/ethereum/ethereum-org-website/pull/10320#issuecomment-1568888220).

Reply:
Awesome, thanks!